test/unit.test.js

```diff
@@ -32,6 +32,48 @@ for (const unitTest of fs.readdirSync(join(__dirname, 'unit'))) {
     console.log('skipping symlink test on Windows: ' + unitTest);
     continue;
   }
+  if (unitTest === 'tsx2') {
```

Comment:
As far as I can tell, the only difference between this unit test and the others is that there is a `readFileMock`.

So instead of copy/paste, let's keep it [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and instead add a `expect(readFileMock.mock.calls.length).toBe(2);` when the unitTest is the tsx one.

Does that make sense?

Reply:
Yes, it does. I'll make that change and push it up.